recipes/binutils/binutils.inc

```diff
@@ -12,12 +12,12 @@ BUGTRACKER = "http://sourceware.org/bugzilla/"
 
 RECIPE_TYPES = "cross sdk-cross machine canadian-cross"
 
-inherit autotools make-vpath gettext
+inherit autotools make-vpath
 
 require conf/fetch/gnu.conf
 SRC_URI = "${GNU_MIRROR}/binutils/binutils-${PV}.tar.bz2 "
 
-DEPENDS = "native:flex native:bison"
+DEPENDS = "native:flex native:bison native:makeinfo native:m4"
```

Comment:
I think you should add native:makeinfo to default.conf and all.conf in this PR, so we can merge it independently of changes in base.

And creating a PR for texinfo in base would be great.